build_tools/webpack_json.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import json
+import logging
+import os
+import sys
+
+from customize_build import load_plugins_from_file
+
+from kolibri.core.webpack.hooks import WebpackBundleHook
```

Comment:
According to the buildkite error, I think we might need to call `set_env()` before doing this import

Reply:
Have followed the suggestion from `cli.py` to move set_env behaviour into kolibri.__init__, so this is now working, and we should always be able to use kolibri as an imported library without special setup!